ngraph/test/files/paddlepaddle/gen_scripts/generate_yolo_box.py

```diff
@@ -0,0 +1,66 @@
+#
+# pool2d paddle model generator
+#
+import numpy as np
+from save_model import saveModel
+
+def yolo_box(name : str, x, img_size, attrs : dict):
+    import paddle as pdpd
+    pdpd.enable_static()
+    
+    with pdpd.static.program_guard(pdpd.static.Program(), pdpd.static.Program()):
+        node_x = pdpd.static.data(name='x', shape=x.shape, dtype=x.dtype)
+        node_img_size = pdpd.static.data(name='img_size', shape=img_size.shape, dtype=img_size.dtype)
+        boxes, scores = pdpd.vision.ops.yolo_box(node_x,
+                                                node_img_size,
+                                                anchors=attrs['anchors'],
+                                                class_num=attrs['class_num'],
+                                                conf_thresh=attrs['conf_thresh'],
+                                                downsample_ratio=attrs['downsample_ratio'],
+                                                clip_bbox=attrs['clip_bbox'],
+                                                name=None, 
+                                                scale_x_y=attrs['scale_x_y'])
+
+        cpu = pdpd.static.cpu_places(1)
+        exe = pdpd.static.Executor(cpu[0])
+        # startup program will call initializer to initialize the parameters.
+        exe.run(pdpd.static.default_startup_program())
+
+        outs = exe.run(
+            feed={'x': x, 'img_size': img_size},
+            fetch_list=[boxes, scores])
+        
+        # Save inputs in order of ngraph function, to facilite Fuzzy test, 
+        # which accepts inputs and outputs in this order as well. 
+        saveModel(name, exe, feedkeys=['x', 'img_size'], fetchlist=[boxes, scores], inputs=[x, img_size], outputs=outs)
+
+    return outs
+
+
+def main():
+    # yolo_box
+    pdpd_attrs = {
+            'anchors': [10, 13, 16, 30, 33, 23],
+            'class_num': 2,
+            'conf_thresh': 0.5,
+            'downsample_ratio': 32,
+            'clip_bbox': False, #There is bug in Paddle2ONN where clip_bbox is always ignored.
```

Comment:
Generate test model with clip_bbox = True to cover more code.
Also please remove ONNX file from PR, looks like this generator script doesn't convert it to onnx, which is correct

Reply:
@ceciliapeng2011 can we support clip_bbox=True ? Or we need add a TODO in our implementation